### PR TITLE
First "real" implementation

### PR DESCRIPTION
**Pull request by juergenhemelt**
.. with sync of Schemas between Schema Registry and Egeria incl. mapping of schemas to KafkaTopics in Egeria.

### CODE REVIEW

**Comment by davidradl** [comment]
@juergenhemelt thanks for submitting. It looks like sonatype has identified 4 things to change.I will look through the code and review it more thoroughly today

File: README.md
Comment (davidradl) [inline README.md:6]: I suggest removing

File: README.md
Comment (davidradl) [inline README.md:6]: I am not sure where you got these 3 lines from - could you direct me to any documentation that mentiones them please - as I have not added them to my readmes.

File: README.md
Comment (davidradl) [inline README.md:11]: is this specific to Confluent Schema registry - if so then it should be in the connector name. I suspect we could have other event schema connectors for other technologies in the future

File: build.gradle
Comment (davidradl) [inline build.gradle:37]: I suggest using 3.12 the latest Egeria

File: src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/connection/ConfluentRestCalls.java
Comment (davidradl) [inline src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/connection/ConfluentRestCalls.java:27]: not Atlas.

File: src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/connection/ConfluentRestCalls.java
Comment (davidradl) [inline src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/connection/ConfluentRestCalls.java:53]: maybe create an issue for this and the other exception processing  to track separately

File: src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/connection/ConfluentRestCalls.java
Comment (davidradl) [inline src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/connection/ConfluentRestCalls.java:94]: maybe change version to schemaVersion and javadoc

File: src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/connection/ConfluentRestCalls.java
Comment (davidradl) [inline src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/connection/ConfluentRestCalls.java:91]: this looks incorrect I think it is returing a schema.

File: build.gradle
Comment (davidradl) [inline build.gradle:42]: I think we need a settings.gradle file to supply the root project name. Unless there is some other way to do this.

File: src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/EventSchemaIntegrationProvider.java
Comment (davidradl) [inline src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/EventSchemaIntegrationProvider.java:21]: I think the name of this connector should include Confluent.

File: build.gradle
Comment (davidradl) [inline build.gradle:34]: the ext.name and description above should refer to this connector

File: README.md
Comment (planetf1) [inline README.md:2]: this is the old label - we dropped ODPi in newer statements

File: README.md
Comment (planetf1) [inline README.md:6]: we don't use azure builds any more, just github

File: README.md
Comment (planetf1) [inline README.md:7]: we're not using Sonar fully either - and not for this repo

File: README.md
Comment (planetf1) [inline README.md:8]: If this repo produces a single artifact, you could point it specifically to the correct name - this above though is core egeria.

File: README.md
Comment (planetf1) [inline README.md:46]: I think we should drop the ODPi here too

File: build.gradle
Comment (planetf1) [inline build.gradle:55]: this should be 'compileOnly' - for all of the components we think are already in the chassis. It's more specific, and would avoid the libraries being picked up to include in any uber jar

File: build.gradle
Comment (planetf1) [reply build.gradle:34]: it's worth checking the generated manifest for the maven artifact. The group is specified in the top level gradle. Names can default to directory name, but as you mention settings etc can override. Worth a second iteration once this is merged

File: build.gradle
Comment (planetf1) [inline build.gradle:145]: use https on the web url Also this URL is incorrect (connection->connector)

File: src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/EventSchemaIntegrationConnector.java
Comment (planetf1) [inline src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/EventSchemaIntegrationConnector.java:2]: We should check our contribution rules for the project. We've used a common odpi contributors copyright with all code going into open source so far.

We need to check this before merging in .

Comment (planetf1) [reply src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/EventSchemaIntegrationConnector.java:2]: This occurs in other files too - but I won't repeat.

**Review by planetf1 — CHANGES_REQUESTED** [review]

**Comment by planetf1** [comment]
I noticed lift had a few findings
 * there are some printStackTrace() calls, which are generally regarded as potentially insecure - they will likely show up as issues when we run code scans - however I see you have TODO tags, so presume this is known, and will be looked at later. Given they will cause scan warnings I think it may be worth opening up an issue to ensure the change isn't forgotten
 * There were a few unsynchronized method warnings around the connector management. Not looked at code, but could be worth checking those closely

**Comment by planetf1** [comment]
Note - Lift will block the merge with some of it's findings. Expand out the 'commands' section, to add comments which will cause lift to mark the issue to be ignored.

However they mostly look like valid issues

**Pull request closed without merging**
